lib/MusicBrainz/Server/Controller/Area.pm

```diff
@@ -6,6 +6,7 @@ BEGIN { extends 'MusicBrainz::Server::Controller'; }
 with 'MusicBrainz::Server::Controller::Role::Load' => {
     model           => 'Area',
     relationships   => {
+        all         => ['aliases'],
```

Comment:
Instead of loading all relationships, we should probably write a more efficient query that just selects the distinct credit  strings from the DB.

Reply:
Did that now.